narwhal/consensus/src/metrics.rs

```diff
@@ -36,13 +36,22 @@ pub struct ConsensusMetrics {
     /// The time it takes for a certificate from the moment it gets created
     /// up to the moment it gets committed.
     pub certificate_commit_latency: Histogram,
-    /// When a certificate is received on an odd round, we check
-    /// about the previous (even) round leader. We do have three possible cases which
-    /// are tagged as values of the label "outcome":
-    /// * not_found: the leader certificate has not been found at all
-    /// * not_enough_support: when the leader certificate has been found but there was not enough support
-    /// * elected: when the leader certificate has been found and had enough support
+    /// On every even round we expect a leader to be elected and committed. However, this is not
+    /// always the case and this metric gives more insight. The metric follows the commit path, so
+    /// all the nodes are expected to report the same results. For every leader of each round the
+    /// output can be one of the following:
+    /// * committed: the leader has been found and its subdag will get committed - no matter if the leader
+    /// is committed on its time or not (part of recursion)
+    /// * not_found: the leader has not been found on the commit path and doesn't get committed
+    /// * no_path: the leader exists but there is no path that leads to it
     pub leader_election: IntCounterVec,
+    /// Under normal circumstances every odd round should trigger leader election for its previous
```

Comment:
I think you can either keep the comment here or in `report_leader_on_time_metrics` and just make a reference to it from the other place. This way we only have to maintain the original comment in one place.